### DISCUSSION TITLE
One RS485 adapter for both Inverter and Battery BMS?

### DISCUSSION THREAD

**Discussion by zit501** [kellerza/sunsynk#135] [Q&A] [answered]
Hi there,

I'm only learning about RS485 so this may be a silly question. 

Could I use one USB to RS485 adapter for both the inverter comms as well as the BMS? i.e. parallel all the 485 cables from the inverter and 2 batteries to one adapter?
I ask this because I have two Hubble AM-2's, and I see on the [hubble_lithium_am2 repo](https://github.com/mysystem32/hubble_lithium_am2) that they specifically show that for multiple AM-2's one can simply use one of those passive RJ45 splitters, or even just join all the twisted pairs together on multiple CAT cables. So this leads me to ask if I could then parallel the comms for all 3 devices.

If I could use one USB to RS485 adapter for all 3 devices (inverter and 2 x BMS's), then anther question would be if I could then use a USR-W610 with its single RS485 interface to transmit the data from all 3 devices.

Thanks in advance.

**Comment by kellerza** [comment, marked as answer]
This should be possible. I have just read @mysystem32 s repo and seems that the batteries will be slave/server 1&2, set through the dip switches on the AM-2. As long as you set your sunsynk modbus slave/server id to something else, say 10. You should be able to read everything on the same bus, using a splitter.

I’ve also got two AM-2s and might try this at some point. Not first priority as I’m currently experimenting with the dalai LLaMa

**Reply by zit501** [reply]
Thanks for your answer. In the meantime I've purchased a usr-tcp232-410s which has both 485 and 232 connections. But I might also try this 485 idea on the AM-2's as I can very slightly hear the 232 comms coming from the AM-2's BMS and it might get annoying to me :)
